### PR TITLE
217 rename styled components 

### PR DESCRIPTION
**Pull request by Kodehodekatty**
### What is missing from the codebase?

- need to organize the naming convention of the styled components, by adding styled infront of the name
- this PR contains the styles within component folder, will do pages after.

### Changes made in this pull request:

- have altered the style component names, putting "styled" in the start.
- moved the addpartformfields shared styles into its own file higher up, might make it into a reusable component
- have not altered Wrapper and Container since its self explanatory, could look into making some of the wrappers/container into reusable components at later point.
- deleted some unused/unnecessary styled components
- moved the navigation functions into a hook that contains the logic for navigating across the routes/logging out.

## Checklist

-   [x] **Linting**: Have you run the linter to ensure code consistency?
-   [x] **Code Formatting**: Is the code formatted according to project standards?
-   [x] **Review**: Have you reviewed your own changes to ensure they align with best practices?

### CODE REVIEW

File: src/components/Button/Button.tsx [hunk 1]
```diff
@@ -44,17 +44,19 @@ export const Button: FunctionComponent<ButtonProps> = ({
...
-            height={height}
-            type={type}
-            onClick={onClick}
-            form={form}
-            id={id}
-            variant={variant}
-        >
-            {children}
-        </SubmitButton>
+        <Wrapper>
```
Comment (bjornole-kodehode) [inline src/components/Button/Button.tsx:47]: Needed here?

File: src/components/AddPartFormFields/Description/Description.tsx [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
+import { StyledDiv, StyledErrorP } from '../styles';
+import { StyledInputWrap } from './styles';
 
 export const Description = () => {
     const { register } = useFormContext();
     return (
         <StyledDiv>
-            <InputWrap>
+            <StyledInputWrap>
                 <label htmlFor="description">Description </label>{' '}
```
Comment (bjornole-kodehode) [inline src/components/AddPartFormFields/Description/Description.tsx:13]: - fjern {' '}

Comment (bjornole-kodehode) [inline src/components/AddPartFormFields/Description/Description.tsx:13]: - fjern '{' '}'

File: src/components/AddPartFormFields/Description/Description.tsx [hunk 3]
```diff
@@ -1,19 +1,21 @@
 import { ErrorMessage } from '@hookform/error-message';
 import { useFormContext } from 'react-hook-form';
 import { StyledTextArea } from '../Comment/styles';
-import { ErrorP, InputWrap, StyledDiv } from './styles';
+
```
Comment (bjornole-kodehode) [inline src/components/AddPartFormFields/Description/Description.tsx:4]: - remove unnecessary line break?

File: .eslintrc.cjs [hunk 4]
```diff
@@ -22,6 +22,12 @@ module.exports = {
         'react-hooks/exhaustive-deps': 'off', // turning it off for now
         'react/react-in-jsx-scope': 'off',
         'react/prop-types': 'off',
+        'prettier/prettier': [
```
Comment (Benjamin-Lyon) [inline .eslintrc.cjs]: remove this setting and set end of line to LF in your IDE

**Review by Benjamin-Lyon — APPROVED** [review]

**Pull request merged**